### PR TITLE
feat(tag): implement new specs for tag - INNO-1201

### PR DESCRIPTION
**Pull request by yhuard**
- Implements tag component
- Lots of code clean-up

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 94d842803b7f37ab7786ea422498b761a9d517af

https://deploy-preview-804--europa-component-library.netlify.com

File: src/systems/ec/specs/components/tag/docs/code.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    type="button"
+  />
+</Playground>
+
+## Link
+
+<Playground playgroundLink="/storybook/ec/index.html?selectedKind=Tag&selectedStory=default&stories=0">
+  <Tag
+    label="Label"
+    href="/example.html#tag"
```
Comment (degliwe) [inline src/systems/ec/specs/components/tag/docs/code.mdx]: should be example#tag

File: src/systems/ec/specs/components/tag/docs/code.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Playground from '@ecl/website-components/Playground'
+import Tag from '@ecl/ec-react-component-tag/Tag'
+
+## Button
+
+<Playground playgroundLink="/storybook/ec/index.html?selectedKind=Tag&selectedStory=default&stories=0">
```
Comment (degliwe) [inline src/systems/ec/specs/components/tag/docs/code.mdx]: wrong url

File: src/systems/ec/specs/components/tag/docs/code.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <Tag
+    label="Label"
+    href="/example.html#tag"
+  />
+</Playground>
+
+## Facet close
+
+<Playground playgroundLink="/storybook/ec/index.html?selectedKind=Tag&selectedStory=default&stories=0">
```
Comment (degliwe) [inline src/systems/ec/specs/components/tag/docs/code.mdx]: wrong url

File: src/systems/ec/implementations/react/packages/tag/stories/index.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  Default: '',
+  'Facet close': 'facet-close',
+};
+
+storiesOf('Tag', module)
+  .addDecorator(withKnobs)
+  .add('default', () => (
+    <Tag
+      label={text('Label', 'Label')}
+      href={text('Link', '#')}
```
Comment (degliwe) [inline src/systems/ec/implementations/react/packages/tag/stories/index.jsx]: href is breaking storybook, should be `/example#tag`

**Review by degliwe — CHANGES_REQUESTED** [review]

**Review by degliwe — APPROVED** [review]

**Pull request merged**